README.adoc

```diff
@@ -4,6 +4,9 @@ ifdef::env-github,env-browser[:outfilesuffix: .adoc]
 
 
 https://travis-ci.org/CS2103AUG2017-W11-B2[image:https://travis-ci.org/CS2103AUG2017-W11-B2/main.svg?branch=master["Build Status", link="https://travis-ci.org/CS2103AUG2017-W11-B2/main"]]
+https://ci.appveyor.com/project/ZY-Ang/main[image:https://ci.appveyor.com/api/projects/status/2yh61xpwvhmdgdkh?svg=true[Build status]]
+https://coveralls.io/github/CS2103AUG2017-W11-B2/main?branch=master[image:https://coveralls.io/repos/github/CS2103AUG2017-W11-B2/main/badge.svg?branch=master[Coverage Status]]
+https://www.codacy.com/app/ZY-Ang/main?utm_source=github.com&utm_medium=referral&utm_content=CS2103AUG2017-W11-B2/main&utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/fe8643b566224b20b21c4c5bfbb64ae6[Codacy Badge]]
```

Comment:
Did you change something here? There shouldn't be any changes here that are logged if branch started from root of the GeneralDoc repo

Reply:
Err, as in my commits shouldn't be in your PR